_haikus/T-shirt.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: T-shirt!
+author: Raman
+---
+
+Haikus are great fun
+So is haiku based hacking
```

Comment:
Add a `<br>` tag or two spaces `  ` at the end of this line to force a line break in markdown.

Reply:
done